packages/notifi-axios-adapter/lib/mutations/createSupportConversationImpl.ts

```diff
@@ -16,15 +16,16 @@ mutation createSupportConversation {
     }
     name
     createdDate
-     participants {
-      conversationParticipantType
-       profile {
-         id
-          preferredAddress
-          preferredName
-       }
-       resolvedName
-     }
+    participants {
+    conversationParticipantType
+      profile {
+        id
+        preferredAddress
```

Comment:
I think it should use AvatarData and AvatarDataType here, right? It just happens that backgroundImageUrl is the same :)

Reply:
update to use AvatarData as the url link for support staff avatar